src/abilities/Abolished.js

```diff
@@ -108,10 +107,8 @@ export default (G) => {
 				});
 			},
 			activate(path, args) {
-				let ability = this;
```

Comment:
`eslint-typescript` rules don't allow references to `this` when arrow functions can do the job.